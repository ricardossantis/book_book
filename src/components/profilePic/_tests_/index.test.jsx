import React from "react";
import renderer from "react-test-renderer";
import ProfilePic from "../index.jsx";

const userInfo = {
  user: { image_url: "female" },
};

describe("Render Test", () => {
  it("renders a picture", () => {
    const tree = renderer.create(<ProfilePic userInfo={userInfo} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
