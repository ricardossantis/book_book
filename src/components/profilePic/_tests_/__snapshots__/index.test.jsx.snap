// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Test renders a picture 1`] = `
<div
  className="sc-bdnylx iUfuel"
>
  <img
    alt="profile-pic"
    src="cardProfile.svg"
  />
</div>
`;
